jwk/jwk.go

```diff
@@ -0,0 +1,25 @@
+// Package jwk provides the JSON Web Tokens API.
+package jwk
```

Comment:
Indeed. I've opted for a singular term for all packages so far and thought I'd continue the same pattern. Happy to change it though if you think it's a problem.

Reply:
Since "jwks" is not a plural term but an acronym, I think the singular/plural rationale (which I'm all for!) does not apply here, right? If so, I tend to agree with George's suggestion.

Reply:
Yes, initially I thought I'd treat this as a "JSON Web Key" API (hence jwk). 

I guess everybody else expects the "JSON Web Key Set" API, so I've changed it.